src/Frame.dfy

```diff
@@ -7,82 +7,82 @@ module FrameFormat {
   import opened Cipher
 
   datatype Frame = Frame(
-    /* The type of frame in question */
-    typ: FrameType,
-    /* The frame sequence number. For nonframed bodies, this should be 1 */
-    sequence_number: nat,
-    /* A cursor to space for the IV in the ciphertext buffer */
-    iv: ByteBuf,
-    /* A cursor to space for the ciphertext in the ciphertext buffer */
-    ciphertext: ByteBuf,
-    /* A cursor to space for the AEAD tag in the ciphertext buffer */
-    authtag: ByteBuf
-  )
+                     /* The type of frame in question */
+                     typ: FrameType,
+                     /* The frame sequence number. For nonframed bodies, this should be 1 */
+                     sequence_number: nat,
+                     /* A cursor to space for the IV in the ciphertext buffer */
+                     iv: ByteBuf,
+                     /* A cursor to space for the ciphertext in the ciphertext buffer */
+                     ciphertext: ByteBuf,
+                     /* A cursor to space for the AEAD tag in the ciphertext buffer */
+                     authtag: ByteBuf
+                   )
```

Comment:
This seems strange

Reply:
You can add a newline before `Frame(` if you want less whitespace (and I would recommend that, until the formatter can do it itself).
Here it aligns the closing ")" with the "Frame(" and indents arguments by 2 spaces.